jbi/jira/service.py

```diff
@@ -213,7 +215,7 @@ def create_jira_issue(
         fields: dict[str, Any] = {
             "summary": bug.summary,
             "issuetype": {"name": issue_type},
-            "description": description[:JIRA_DESCRIPTION_CHAR_LIMIT],
+            "description": markdown_to_jira(description[:JIRA_DESCRIPTION_CHAR_LIMIT]),
```

Comment:
I wonder what happens if we cut off a comment in the middle of some markup, like `[some link](https://www.ex`, and then that's the character limit. Will `pandoc` handle it? Throw some exception about malformed input? I wonder if we need a smarter truncation of the comment so we break on the last complete word before the character limit 🤔

Reply:
you are right, there is a broader question here, how does pandoc handle malformed input! 

truncating at the last complete word is a good idea, I'll have a look what I can do.